buildscripts/grpc-java-releasing/Dockerfile

```diff
@@ -1,5 +1,14 @@
 FROM protoc-artifacts:latest
 
+# Work around the docker cache. Not really needed
+# because Kokoro instances start off with a blank
+# slate. But useful to have this here because we may
+# want to run these files locally.
+RUN echo "cache buster 4/24/2018"
+
+# protoc-artifacts base image is ancient and curl does
+# not work properly.
+RUN yum -y update; yum clean all || true
```

Comment:
Why isn't this `yum -y update && yum clean all`?

Reply:
No, `RUN yum -y update; yum clean all` would have been the way. Each RUN creates a new layer. Doing the clean in a separate layer is useless.

Reply:
done